index.bs

```diff
@@ -304,6 +308,8 @@ or else the <a>top-level browsing context</a>'s <a>document</a> node
 (referred to as the <dfn for="IntersectionObserver">implicit root</dfn>) if
 the {{IntersectionObserver/root}} attribute is <code>null</code>.
 
+An {{Element}} is defined as having an <dfn for="IntersectionObserver">overflow clip</dfn> if its computed style has a value other than "visible" for its <a>overflow</a>, <a>overflow-x</a>, or <a>overflow-y</a> property.
```

Comment:
"overflow clip" seems quite an odd term because `clip` is a valid value of the `overflow` property, fwiw.

There's no need to mention the overflow property explicitly, as it's just a shorthand of `overflow-x` and `overflow-y`.

Reply:
I changed this to "content clip".